src/NServiceBus.Core/EndpointConfiguration.cs

```diff
@@ -216,23 +196,11 @@ internal InitializableEndpoint Build()
             }
 
             Settings.SetDefault("TypesToScan", scannedTypes);
-            Settings.Set("Endpoint.SendOnly", sendOnly);
```

Comment:
(the default value is already set here: https://github.com/Particular/NServiceBus/pull/4002/files#diff-eacabae3270e571c2781c82ebf4cf1e6R46 )